### ISSUE TITLE
php 5.3 compatibility

### ISSUE THREAD

**Issue by chilek** [tecnickcom/tc-lib-barcode#18]
Really doesn't it work with PHP 5.3?
If it doesn't, would it be difficult to adjust it to 5.3?
Many enterprise linux distributions still use PHP 5.3.

**Comment by nicolaasuni** [comment]
The 5.3 version of PHP is unsupported, its end of life was more than 2 yeas ago (14 August 2014)!
If you are running PHP 5.3 in production I strongly recommend to upgrade to a recent version.
Ref. https://secure.php.net/eol.php

**Comment by chilek** [comment]
@nicolaasuni I know that but RHEL/CentOS 6.x still use PHP 5.3 series maintained by their own!
Fragment of changelog from their php rpm package:

```
* Mon Jul 25 2016 Remi Collet <rcollet@redhat.com> - 5.3.3-48
- don't set environmental variable based on user supplied Proxy
  request header CVE-2016-5385

* Wed Dec 09 2015 Remi Collet <rcollet@redhat.com> - 5.3.3-47
- fix wrong warning in openssl_encrypt() for missing IV
  when IV is not required #1260315
- fix segfault's when you try and allocate an SplFixedArray
  with size >= 9999 #1071344
- segfault in php_pgsql_meta_data CVE-2015-4644  #1234434
- add options to enable TLS in curl #1255920
- fix segfault in gc_collect_cycles #1122681

* Fri Jul 03 2015 Remi Collet <rcollet@redhat.com> - 5.3.3-46
- fix gzfile accept paths with NUL character #1213407
- fix patch for CVE-2015-4024

* Wed Jun 10 2015 Remi Collet <rcollet@redhat.com> - 5.3.3-45
- fix more functions accept paths with NUL character #1213407

* Mon Jun 08 2015 Remi Collet <rcollet@redhat.com> - 5.3.3-44
- soap: missing fix for #1222538 and #1204868
```

are many adjustments in code needed to make it PHP 5.3 compatible? Maybe I could help with it anyway?
